tox.ini

```diff
@@ -1,9 +1,14 @@
 [tox]
-envlist = py27,py35
+envlist = py{27,35},django18
```

Comment:
We should also test this with Django 1.11

Reply:
Good point.